editor/src/clj/editor/workspace.clj

```diff
@@ -556,8 +556,8 @@ ordinary paths."
          :actions [{:text "Open game.project"
                     :on-action #(ui/execute-command
                                   (ui/contexts (ui/main-scene))
-                                  :open
-                                  {:resources [(find-resource workspace "/game.project")]})}]})
+                                  :file.open
+                                  "/game.project")}]})
```

Comment:
I'm already getting excited about exposing command invocation to editor scripts...